vmconf.py

```diff
@@ -13,4 +13,47 @@
 # limitations under the License.
 
 """Stores various configuration options and constants for Oppia-ml."""
-pass
+
+# The platform for the storage backend. This is used in the model-switching
+# code in core/platform.
+PLATFORM = 'gce'
+
+# Whether we should serve the development or production experience.
+# DEV_MODE should only be changed to False in the production environment.
+DEV_MODE = True
+
+# Default communication URL to be used to communicate with Oppia server.
+# This URL will be used in local development.
+DEFAULT_COMMUNICATION_URL = 'https://localhost'
+
+# Default communication PORT to be used to communicate with Oppia server.
+# This URL will be used in local development.
+DEFAULT_COMMUNICATION_PORT = 8181
+
+# Communication URL which is to be used in production environment to
+# communicate with Oppia server.
+SERVER_COMMUNICATION_URL = 'https://www.oppia.org'
+
+# Communication PORT which is to be used in production environment to
+# communicate with Oppia server.
+SERVER_COMMUNICATION_PORT = 80
+
+# Default ID which is to be used in local development environment.
+DEFAULT_VM_ID = 'vm_default'
+
+# Default shared key which is to be used in local development environment.
+DEFAULT_VM_SHARED_SECRET = '1a2b3c4e'
+
+# Name of metadata parameter which will contain ID of the VM in production
+# environment in GCE.
+METADATA_VM_ID_PARAM_NAME = 'vm_id'
```

Comment:
Yeah Google does provide `ID` to each instance and I don't think it will be possible to use that ID because we also need to add `vm_id` in Oppia's admin page so that access is allowed. But [Google's](https://cloud.google.com/compute/docs/storing-retrieving-metadata#default) metadata page says that we can only retrieve default metadata from within instance (and `ID` is default). So it is hard to obtain it. It also lessens the flexibility because admin can't give user friendly VMIDs (It is easier to remember them).